src/dispatching/dialogue/storage/trace_storage.rs

```diff
@@ -0,0 +1,55 @@
+use std::{fmt::Debug, sync::Arc};
+
+use futures::future::BoxFuture;
+use log::trace;
+
+use super::Storage;
+
+/// Storage wrapper for logging purposes
+///
+/// Reports about any dialogue update or removal action on `trace` level
```

Comment:
The doc probably can be made better some rephrasing + noting that `log` crate is used may be good

Reply:
Quite unsure what else needed to be stated in documentation, except the fact that `log` crate is being used